sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -124,10 +124,39 @@ def __init__(
         self.require_encryption = kwargs.get("require_encryption", False)
         self.key_encryption_key = kwargs.get("key_encryption_key")
         self.key_resolver_function = kwargs.get("key_resolver_function")
-        self._config, self._pipeline = self._create_pipeline(
-            self.credential, storage_sdk=service, **kwargs
+
+        self._configure_credential(self.credential)
+
+        self._config = kwargs.get("_configuration") or AzureTableConfiguration(
+            url=self.url,
+            headers_policy=StorageHeadersPolicy(**kwargs),
```

Comment:
i see you're passing in your own list. of policies because you're adding policies that configuration doesn't have properties for. In that case, I wouldn't bother wit all of this `Configurationi` creation, since you're just going to override all of. this configuration policy information with the policies you have defined under `self._policies`. I would instead just use. the generate code's configuration